agent/containers/images/visualizers/DockerfileLive

```diff
@@ -19,16 +19,13 @@ ADD combo.json .
 ## THE FOLLOWING LINE WAS PARTIALLY TAKEN FROM https://github.com/grafana/grafana-docker/blob/master/Dockerfile#L7#L13
 ENV PATH=/usr/share/grafana/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin \
```

Comment:
I'd just take all of these out of the Dockerfile.

Reply:
These are not used in run.py though, and are necessary for grafana. Should the env vars be simply defined in run.py instead?